content/swiftui/concepts/viewmodifier/terms/foregroundColor/foregroundColor.md.md

```diff
@@ -0,0 +1,56 @@
+﻿---
+Title: `.foregroundColor()`
+Description: The `.foregroundColor()  modifier sets the foreground color displayed in a view`.
```

Comment:
Hello, would you mind please removing the backticks on these lines and replacing them with single quotes?

Also, if you could contain the full description sentence on line 3 inside the quotes, that would be appreciated.

Reply:
Thank you. I have updated the file and committed.